diffedit.md

```diff
@@ -552,201 +460,52 @@ for i, t in enumerate(tqdm(denoising_scheduler.timesteps[start_step:])):
 
 
 ```python
-caption = "a bowl of fruits"
-query = "a bowl of pears"
+caption = "bowl of fruits"
+query = "sports cars"
 
-diffedit(fruit_bowl_image, caption, query, noise_strength=1)
+diffedit(fruit_bowl_image, caption, query, noise_strength=1, seed=1)
 ```
 
 
-      0%|          | 0/50 [00:00<?, ?it/s]
-
-
 
     
-![png](diffedit_files/diffedit_92_1.png)
+![png](diffedit_files/diffedit_99_1.png)
     
 
 
 
     
-![png](diffedit_files/diffedit_92_2.png)
+![png](diffedit_files/diffedit_99_2.png)
     
 
 
-We can see a very nice mask generation here, along with some slightly wonky looking pears, that's mainly an issue with our diffusion model's image generation in this case rather than anything that we've done specifically, so we'll accept slightly wonky pears for now.
-
-## Limitations
-
-Something else which is worth considering is the limitations of the DiffEdit mask creation technique. If there are huge differences between the original image caption and the query string then the mask that is generated is not as accurate, which has the knock-on effect of leading to poor final image generation. Take our example below, we'll use the same original image and caption as before, but this time use the string `"bowl of cars"` as the query. As we can see the mask creation includes random areas of the background due to the lack of similarities with the original image and the intended query, and generates a rather odd final image.
-
-
-```python
-caption = "bowl of fruits"
-query = "bowl of cars"
-
-diffedit(fruit_bowl_image, caption, query, noise_strength=1)
-```
+## Conclusion
 
+When working on this DiffEdit pipeline we set out to implement the techniques in the paper using our own custom pipeline. As you can see in the example above, we were able to acheive this with accurate mask generation and decent final image replacement. This shows that DiffEdit is a very powerful technique, and it was suprisingly easy to adapt an img2img pipeline to work with the DiffEdit technique. The ability to pass in just two different text prompts and have an accurate mask generated as a result was truly impressive.
 
-      0%|          | 0/50 [00:00<?, ?it/s]
+There is still more for us to explore with this technique, from tweaking the various parameters further to tune both the mask creation and the image generation, to further understanding what prompts give the best results. It would also be interesting to adapt this to work with Stable Diffusion 2 or SDXL and see how that affects both the mask creation and final image generation.
 
+TODO: maybe talk about google magic eraser?
```

Comment:
Would be interesting though I think mention of SDXL and Stable Diffusion 2 may be sufficient.